frontend/public/style/_overrides.scss

```diff
@@ -512,3 +512,357 @@ kbd {
     display: none;
   }
 }
+
+// temporary td width's for demo only, responsive pf4 css to follow
+@media (min-width: $screen-md-min){
+  .pf-c-table .meta-status {
+    min-width: 100px;
+  }
+}
+
+// pf4 table spacings adjusted to match existing styles
+.pf-c-table tr > * {
+  --pf-c-table-cell--PaddingLeft: 16px;
+  --pf-c-table-cell--PaddingRight: 16px;
+  --pf-c-table--m-compact-cell--first-last-child--PaddingLeft: 16px;
+  --pf-c-table--m-compact-cell--first-last-child--PaddingRight: 16px;
+}
+
+.pf-c-table.pf-m-compact .pf-c-table__action {
+  --pf-c-table__action--PaddingLeft: 16px;
+  --pf-c-table__action--PaddingRight: 16px;
+  --pf-c-table__action--PaddingTop: 8px;
+  --pf-c-table__action--PaddingBottom: 8px;
+
+  .pf-c-dropdown__toggle {
+    font-size: $font-size-base;
+  }
+  .pf-c-dropdown__menu-item {
+    font-size: $font-size-base;
+  }
+}
+
+// pf table overrides
+.pf-c-table {
```

Comment:
all of the following pf-c-table CSS is currently in review and can potentially be added to PF4 Core. cc: @christiemolloy